src/mmrelay/db_utils.py

```
@@ -519,39 +817,141 @@ def wipe_message_map():
+        Parameters:
+            cursor (sqlite3.Cursor): Cursor used to execute the deletion.
+        """
         cursor.execute("DELETE FROM message_map")
-        conn.commit()
-    logger.info("message_map table wiped successfully.")
+
+    try:
+        manager.run_sync(_wipe, write=True)
+        logger.info("message_map table wiped successfully.")
+    except sqlite3.Error as e:
+        logger.error("Failed to wipe message_map: %s", e)
```

Ruff (0.14.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)